auv_sim/auv_sim_bringup/launch/inc/start_world.launch.xml

```diff
@@ -2,7 +2,7 @@
 <launch>
   <arg name="gui" default="true" />
   <arg name="debug" default="false" />
-  <arg name="world_name" default="$(find auv_sim_description)/worlds/seabed.world" />
+  <arg name="world_name"/>
```

Comment:
We already specify the default world in the main launch file. Why choose a default one here again? Deleted.